packages/subgraph/src/utils.ts

```diff
@@ -277,3 +277,22 @@ export function getOrder(
     return blockNumber.times(ORDER_MULTIPLIER).plus(logIndex);
 }
 
+/**************************************************************************
+ * Log entities util functions
+ *************************************************************************/
+
+export function createLogID(
```

Comment:
It would have been neat if this incremented based on account updates.

Reply:
The database would automatically order the ID-s a bit better, yes, but it depends on how the database is set up, regarding how it orders "-5" vs "-12". If it prioritizes shorter length then the ID-s would be perfectly sorted at the time of insertion, yes. This can have some query performance implications.

The other small advantage is that it's easy to see how many updates the account has had. This might make planning for queries easier in some cases... Also could be useful information in general.

I'm not saying this is a big deal at all... Just discussing and saying an indexer based ID would have felt more natural to me. :)

Reply:
What do you mean by indexer-based ID?

We could append an order id as part of the ID (generated based on https://github.com/superfluid-finance/protocol-monorepo/pull/805).

Reply:
I mean like id-1, id-2, id-3, id-4, id-5... etc after every change.

Reply:
Do you want it to store in memory or a database?

Reply:
Yes, why not. 

Sure let's add one more property to track updates count, please create a separate ticket. Also regarding `triggered event: Event!` Please mention what properties you want to keep in the HOL entity.

Reply:
One problem is, that the code is pretty much scattered updating from multiple places and due to which maybe be counter or version possibly need to pass a parameter.